Cargo.lock

```diff
@@ -338,7 +338,7 @@ dependencies = [
 
 [[package]]
 name = "cargo-platform"
-version = "0.1.1"
+version = "0.1.0"
```

Comment:
I think something went wrong during rebasing...

Reply:
I was using the magical rebase and rustfmt each commit command from zulip, but that always picks my version on conflicts -.-

Reply:
Fixed, but I better go recheck all my other prs